File: app/src/main/java/io/treehouses/remote/pojo/CommandListItem.java
package io.treehouses.remote.pojo;

public class CommandListItem {
    String title, command;

    public CommandListItem(String title, String command) {
        this.title = title;
        this.command = command;
    }

    public CommandListItem() {
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getCommand() {
        return command;
    }

    public void setCommand(String command) {
        this.command = command;
    }
}

File: app/src/main/java/io/treehouses/remote/pojo/CommandListItem.kt
package io.treehouses.remote.pojo

class CommandListItem (private var title: String, private var command:String) {

    fun getTitle(): String {
        return title
    }

    fun setTitle(title: String) {
        this.title = title!!
    }

    fun getCommand(): String {
        return command
    }

    fun setCommand(command: String?) {
        this.command = command!!
    }

}